conf/services.conf

```diff
@@ -0,0 +1,163 @@
+# Use Simple API to create endpoints for publicly accessible research frontend.
+simple_api_endpoints = {
+    # Library
+    library_search = {
+        type = search,
+        table = ca_objects,
+        restrictToTypes = [ library ],
+        checkAccess = [ 1 ],
+        content = {
+            object_id = ^ca_objects.object_id,
+            idno = ^ca_objects.idno,
+            Title = ^ca_objects.preferred_labels,
+            Author = ^ca_objects.Author,
+            Publisher = ^ca_objects.Publisher,
+            DateOfPublication = ^ca_objects.DateOfPublication,
+            PlaceOfPublication = ^ca_objects.PlaceOfPublication,
+            PublicationType = ^ca_objects.PublicationType,
```

Comment:
Pretty sure it comes with the actual value not the id so it's okay. The multivalued fields come out as semicolon separated lists of names which I can process client side. Creator is an interesting one.